rustls/src/client/builder.rs

```diff
@@ -163,6 +163,34 @@ impl ConfigBuilder<ClientConfig, WantsClientCert> {
         Ok(self.with_client_cert_resolver(Arc::new(resolver)))
     }
 
+    /// Sets a raw public key and matching private key for use
+    /// in client authentication.
+    ///
+    /// `key_der` is a DER-encoded private key as PKCS#1, PKCS#8, or SEC1. The
```

Comment:
"DER-encoded private key as PKCS#1, PKCS#8, or SEC1" reads a little awkwardly to me. Maybe "private key encoded as PKCS#1, PKCS#8 or SEC1 DER"?